tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.cluster import get_pg_balancer_status
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-2231")
+class TestCephDefaultValuesCheck(ManageTest):
+
+    def test_ceph_default_values_check(self):
+        """
+        This test checks ceph default values taken from OCS 4.5 with the
+        current values in the cluster
+
+        """
+        # The default ceph osd full ratio values
+        expected_full_ratios = {
+            'full_ratio': 0.85,
+            'backfillfull_ratio': 0.8,
+            'nearfull_ratio': 0.75
+        }
+        actual_full_ratios = {}
+        ct_pod = pod.get_ceph_tools_pod()
+        log.info("Checking the values of ceph osd full ratios in osd map")
+        osd_dump_dict = ct_pod.exec_ceph_cmd('ceph osd dump')
+        for ratio_parm, value in expected_full_ratios.items():
+            ratio_value = osd_dump_dict.get(ratio_parm)
+            actual_full_ratios.update(
+                {ratio_parm: float(round(ratio_value, 2))}
+            )
+            if not float(round(ratio_value, 2)) == value:
+                log.error(
+                    f"Actual {ratio_parm} value is {ratio_value:.2f} NOT "
+                    f"matching the expected value {value}"
+                )
+        assert expected_full_ratios == actual_full_ratios, (
+            f"Actual {actual_full_ratios} values does not match "
+            f"expected full ratio values {expected_full_ratios}"
+        )
```

Comment:
Dumping dictionaries into the message of this assert is not necessary. Pytest provides quite clear report already, and having dictionaries again in the message doesn't add any value and could be actually puzzling at first:

```
>       assert expected_full_ratios == actual_full_ratios, (
            f"Actual {actual_full_ratios} values does not match "
            f"expected full ratio values {expected_full_ratios}"
        )
E       AssertionError: Actual {'full_ratio': 0.8500000238418579, 'backfillfull_ratio': 0.800000011920929, 'nearfull_ratio': 0.75} values does not match expected full ratio values {'full_ratio': 0.85, 'backfillfull_ratio': 0.8, 'nearfull_ratio': 0.75}
E       assert {'backfillful..._ratio': 0.75} == {'backfillful..._ratio': 0.75}
E         Omitting 1 identical items, use -vv to show
E         Differing items:
E         {'backfillfull_ratio': 0.8} != {'backfillfull_ratio': 0.800000011920929}
E         {'full_ratio': 0.85} != {'full_ratio': 0.8500000238418579}
E         Full diff:
E           {
E         -  'backfillfull_ratio': 0.8,
E         +  'backfillfull_ratio': 0.800000011920929,
E         ?                           ++++++++++++++
E         -  'full_ratio': 0.85,
E         +  'full_ratio': 0.8500000238418579,
E            'nearfull_ratio': 0.75,
E           }
```

See that esp. the full diff report is quite nice.

Reply:
Done.